app/controllers/dashboard_controller.rb

```diff
@@ -1,14 +1,54 @@
 # frozen_string_literal: true
 
+require 'will_paginate/array'
+
 class DashboardController < ApplicationController
-  # GET /
   def index
     @participants_per_edition = participants_per_edition
     @implication_level = implication_level
+    @average_points_per_edition = average_points_per_edition
+    @first_places_per_edition = first_places.paginate(page: params[:page], per_page: 12)
```

Comment:
will we always have 12 per_page? is this used in multiple place?

If we are not sure this will always be the same and is used in multiple places maybe it's better to have a constant somewhere and use it.

What do you think?

Reply:
10 is the default and is set in a config file, here is something specific, we want editions for the past 12 months